avro/src/error.rs

```diff
@@ -377,6 +377,8 @@ pub enum Error {
     #[error("Failed to compress with flate: {0}")]
     DeflateCompress(#[source] std::io::Error),
 
+    // no longer possible after migration from libflate to miniz_oxide
+    #[deprecated(since = "0.18.0", note = "This error can no longer occur")]
```

Comment:
```suggestion
    #[deprecated(since = "0.19.0", note = "This error can no longer occur")]
```
0.18.0 release is currently being voted at dev@avro.apache.org